### PR TITLE
Move samples to the correct location

### PR DESCRIPTION
**Pull request by gaoning777**
AI-platform: move to the core, restructure the samples
artifact_location: move to the core
volume_op/volume_snapshot_op: keep the basic one that demonstrates the feature while placing the others to the contrib.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1759)

### CODE REVIEW

File: samples/core/ai-platform/README.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ The [Chicago Crime Data](https://console.cloud.google.com/marketplace/details/ci
 ## Assets
 
 * Notebooks
-  * [Chicago Crime Research.ipynb](Chicago%20Crime%20Research.ipynb) Illustrates the process of building a **model** outside of production.
-  * [Chicago Crime Pipeline.ipynb](Chicago%20Crime%20Pipeline.ipynb) Illustrates how to build a **pipeline** and deploy it to a Kubeflow instance
+  * [Chicago Crime Pipeline.ipynb](Chicago%20Crime%20Pipeline.ipynb) Illustrates how to build a **pipeline** and deploy it to a Kubeflow instance.
+  * [/local/Chicago Crime Research.ipynb](Chicago%20Crime%20Research.ipynb) Illustrates the process of building a **model** outside of production.
```
Comment (numerology) [inline samples/core/ai-platform/README.md]: Those links are not actually working. Do we want to keep them anyway or to use the code style?

Comment (gaoning777) [reply samples/core/ai-platform/README.md]: Let me remove the links. Good catch.

Comment (gaoning777) [reply samples/core/ai-platform/README.md]: Done.

Comment (Ark-kun) [inline samples/core/ai-platform/README.md]:
```suggestion
  * [Chicago Crime Research.ipynb](/local/Chicago%20Crime%20Research.ipynb) Illustrates the process of building a **model** outside of production.
```

Comment (gaoning777) [reply samples/core/ai-platform/README.md]: Could you check the new commit? Thanks

**Pull request merged**